packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/Choices/Choices.stories.js

```diff
@@ -0,0 +1,72 @@
+import zooTheme from '@zooniverse/grommet-theme'
+import { Box, Grommet } from 'grommet'
+import React from 'react'
+
+import Choices from './Choices'
+import task from '../../../../../data/task'
```

Comment:
Not ideal file or folder naming and duplicates mock survey task with task model and annotation model. Major refactor (to utilize factory?) can come in subsequent PR, but maybe naming, folder, or data formatting cleanup should be included in this PR?